javascript/central-login-oidc/README.md

```diff
@@ -8,17 +8,15 @@ This instructs the SDK to redirect the user to the login application that uses e
 
 After successful authentication, the SDK redirects the user back to this sample application to obtain OAuth/OIDC tokens and complete the centralized login flow.
 
-To configure this sample, follow the steps in [Getting Started](#getting-started).
-
-Then, run the sample app as follows:
-```
-npm run start:central-login
-``` 
-
 ### Instructions
 
-* [PingOne server](#pingone-server)
+To configure your server, following the steps in [Getting Started](https://docs.pingidentity.com/sdks/latest/sdks/getting-started.html).
 
-#### PingOne server
+Then, create your configuration `.env` file based on the included `.env.example`.
 
-* For instructions on configuring this sample to work with a PingOne server, refer to the [PingOne JavaScript tutorial](https://docs.pingidentity.com/sdks/latest/sdks/tutorials/javascript/index.html#tutorial_steps).
\ No newline at end of file
+Set the `SERVER_TYPE` property to either `PINGONE` or `AIC`, based on the server used.
```

Comment:
@george-bafaloukas-forgerock what does this do? The user should be able to use PingFederate or PingAM as well.

Reply:
You can just add conditionals in `main.js` to achieve this, or add a flag for redirect on logout (and default to false or something).

Reply:
This is how it works in JS, it checks the value from the `env` and conditionally does what it needs on the logout.

All teh configuration on this, and other samples, happens on the `env` file. I would not want to add extra configuration for the developers having to change properties on the JS files as well.

Regarding PingFed and the rest...We can name that `AIC` and `PING`. Basically, if you select `PING` will work with anything on the Ping side. If it's AIC, all the legacy FR products. AIC and AM are one and the same as far as the SDK is concerned

Reply:
Why don't we take away server and have a config item like `RedirectLogout = false` or something?

Reply:
Not against that. But would like to see what @ryanbas21 and @cerebrl think on that. Changing the variable name to anything that makes it more clear for what it does and how is used is easy.

Reply:
Hmm ... I'm not sure developers would know when `RedirectLogout` needs to be `true` or `false`. Rather, they would know if they are using PingOne or AIC. I lean more towards using the product as that feels more obvious to the customer, rather than specific to the programatic requirement.

Reply:
Remember that this fix is taking place soon as well, so you don't have to have redirect logout. At a minimum @george-bafaloukas-forgerock, update this file to accommodate PingFederate as well, which doesn't require redirect, so it can work like AIC. After the fix next sprint (and when this makes it into JS SDK).

Reply:
Resolving this for now. When the JS SDK and Server side is updated we will revisit and change the sample